dspace-api/src/main/java/org/dspace/app/statistics/clarin/ClarinMatomoBitstreamTracker.java

```diff
@@ -122,8 +122,6 @@ public void trackBitstreamDownload(Context context, HttpServletRequest request,
 
         List<Item> items = clarinItemService.findByBitstreamUUID(context, bit.getID());
         if (CollectionUtils.isEmpty(items)) {
-            log.error("Cannot find the Item for the bitstream with ID: " + bit.getID() +
```

Comment:
why?

Reply:
Ondrej wanted to remove a log that could generate noise. https://github.com/ufal/clarin-dspace/issues/1121